install_config/topics/ocp_osp_prerequisites.adoc

```diff
@@ -0,0 +1,35 @@
+=== {product-title} Prerequisites
+
+A successful deployment of {product-title} requires many prerequisites. This consists of
+a set of infrastructure and host configuration steps prior to the actual
+installation of {product-title} using Ansible. In the following subsequent sections, details
+regarding the prerequisites and configuration changes required for an {product-title} on a
+OpenStack environment are discussed in detail.
+
+NOTE: All the OpenStack CLI commands in this reference environment are executed
+using the CLI `openstack` commands within the OpenStack director node. If using a 
+workstation or laptop to execute these commands instead of the OpenStack
+director node, please ensure to install the following packages found
+within the specified repositories.
+
+Example:
+
+Enable the rhel-7-server-openstack-{rhosp_version}-rpms and Ansible repositories
+
+----
+$ sudo subscription-manager repos \
+--enable rhel-7-server-openstack-13-rpms \
+--enable=rhel-7-server-ansible-2.5-rpms
```

Comment:
the official document says that OCP-3.10 use the ansible-2.4, I thinks is better to follow this [suggestion](https://docs.openshift.com/container-platform/3.10/getting_started/install_openshift.html).

Reply:
use this "--enable="rhel-7-server-ansible-2.4-rpms"